app/addons/fauxton/components.react.jsx

```diff
@@ -403,7 +405,7 @@ function (app, FauxtonAPI, React, ReactDOM, ZeroClipboard, ReactBootstrap) {
             {content}
           </Modal.Body>
           <Modal.Footer>
-            <button className="btn btn-success js-btn-success" onClick={this.props.onSubmit}>
+            <button className={btnClasses} onClick={this.props.onSubmit}>
               <i className="fonticon-ok-circled"></i> {this.props.successButtonLabel}
             </button>
```

Comment:
Just FYI, I did try using the ConfirmButton component but ran into a circular reference problem where one file required the other. As discussed, way back when, I'm going to combine those two files + this problem will go away.